### DISCUSSION TITLE
Add exports entries without extension in package.json

### DISCUSSION THREAD

**Discussion by maicol07** [lit/lit#2626] [Ideas]
Hi,
can you add the export entries without extensions? This will help when using ESLint import plugin. Check the relevant discussion: https://github.com/import-js/eslint-plugin-import/issues/2141

**Comment by justinfagnani** [comment]
Allowing extensionless imports would make it difficult to define compact import maps for the Lit packages. We try to make sure that our packages can work with minimal tooling, and import maps are an important piece of consuming modules with bare module specifiers in browsers without any build steps. Import maps support prefix matches, but can't add extensions.

We also couldn't change from extension-required imports to extension-less imports, as that would be a breaking change. We also can't allow more than one way to import a module because that would lead to module duplication.

Is there no way to configure the lint rule to respect the exports of the package you're importing from?

**Reply by maicol07** [reply]
No, only making extensions required will work. However, it would be better without them

**Reply by ljharb** [reply]
Why would it be difficult? Things already must parse "exports"; the presence of the extension doesn't make that more or less difficult.

Why would more than one path pointing to the same module cause duplication? Modules are cached by final filename, not by the specifier that links to it.

**Reply by maicol07** [reply]
@justinfagnani any update?
